tests/Http/Three/SignupTest.php

```diff
@@ -199,6 +299,65 @@ public function testSignupWithIncludedPosts()
         ]);
     }
 
+    /**
+     * Test for retrieving a signup with included rejected post info. as admin and non-admin/non-owner.
+     *
+     * GET /api/v3/signups/186?include=posts
+     * @return void
+     */
+    public function testSignupWithIncludedPostsWithMultipleCredentials()
+    {
+        $signup = factory(Signup::class)->create();
+        $post = factory(Post::class)->create();
+        $post->signup()->associate($signup);
+        $post->status = 'rejected';
```

Comment:
If you want, there's a [factory]() for this!

```php
$post = factory(Post::class, 'rejected')->create();
$signup = $post->signup;

```

Reply:
oh awesome, thank you!!